### ISSUE TITLE
What are the pros and cons of zustand-persist compared to the built in persistance?

### ISSUE THREAD

**Issue by krstffr** [roadmanfong/zustand-persist#8]
Hi!

I was looking for a persistance solution for zustand and found this repo as well as the built in https://github.com/pmndrs/zustand#persist-middleware. What are the pros and cons of `zustand-persist` compared to the built in persistance?

Thanks!

**Comment by Aaike** [comment]
I will just attempt to answer this one. I believe this was created before zustand persist middleware was created (I believe it was added in zustand v3). 
I tried to use the zustand persist middleware in one of my projects but I couldn't because ii was still on zustand v2.

**Comment by gabrielmoncea** [comment]
Yes, this library was created before zustand had it's built-in middleware

**Comment by pvinis** [comment]
maybe this issue can be pinned?
